src/gui/ExportProjectDialog.cpp

```diff
@@ -142,11 +142,11 @@ OutputSettings::StereoMode mapToStereoMode(int index)
 	switch (index)
 	{
 	case 0:
-		return OutputSettings::StereoMode_Stereo;
+		return OutputSettings::StereoMode_Mono;
 	case 1:
-		return OutputSettings::StereoMode_JointStereo;
+ 		return OutputSettings::StereoMode_Stereo;
 	case 2:
-		return OutputSettings::StereoMode_Mono;
+		return OutputSettings::StereoMode_JointStereo;
 	default:
 		return OutputSettings::StereoMode_Stereo;
```

Comment:
Please add an extra tab to everything withing the brackets
```
{
    like
    this
}
```

Reply:
We don't have a rule for `switch` statement, so it's fine.